modules/programs/fish.nix

```
@@ -349,6 +349,18 @@ in {
               config.home.sessionVariables)
           }
 
+          for p in "$HOME/.nix-profile/bin"
+            if not contains $p $PATH
+              set PATH $p $PATH
+            end
+          end
+
+          for p in "$HOME/.nix-defexpr/channels"
+            if not contains $p $NIX_PATH
+              set -xg NIX_PATH $p $NIX_PATH
```

```suggestion
              set -xgp NIX_PATH $p
```

Same question as above.